Skate_ERP_preprocessing_IO.m

```diff
@@ -75,326 +66,77 @@
         EEG = pop_eegthresh(EEG,1,[1:size(EEG.data,1)-2],-200,200,EEG.xmin,EEG.xmax,0,1);
         
-        if i_cond ==1
-        tempEEG1 =   EEG;
-        elseif i_cond==2
-        tempEEG2 = EEG;
-        
-        tempEEG = pop_mergeset(tempEEG1,tempEEG2,1);
-        
-        %now select the corrected trials
-        EEG = pop_selectevent( tempEEG, 'type',5,'renametype','Target','deleteevents','on','deleteepochs','on','invertepochs','off');
-        EEG = pop_editset(EEG, 'setname',[subs{i_sub} '_' exp '_' conds{i_cond} '_Corrected_Target']);
-        EEG = pop_saveset( EEG, 'filename',[subs{i_sub} '_' exp '_facing_in_Corrected_Target.set'],'filepath',[Pathname 'segmentsIO\']);
+        tempEEG = EEG;
         
-        
-        EEG = pop_selectevent( tempEEG, 'type',3 ,'renametype','Standard','deleteevents','on','deleteepochs','on','invertepochs','off');
-        EEG = pop_editset(EEG, 'setname',[subs{i_sub} '_' exp '_' conds{i_cond} '_Corrected_Standard']);
-        EEG = pop_saveset( EEG, 'filename',[subs{i_sub} '_' exp '_facing_in_Corrected_Standard.set'],'filepath',[Pathname 'segmentsIO\']);
-        
-        end
-        
-    end 
-end 
-commandhistory
-%%
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-%                    REGULAR OUT
-%
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-%%
-clear all
-close all
-ccc
+ %now select the corrected trials
+%         EEG = pop_selectevent( tempEEG, 'type',5,'renametype','Target','deleteevents','on','deleteepochs','on','invertepochs','off');
+%         EEG = pop_editset(EEG, 'setname',[subs{i_sub} '_' exp '_' conds{i_cond} '_Corrected_Target']);
+%         EEG = pop_saveset( EEG, 'filename',[subs{i_sub} '_' exp '_' conds{i_cond} '_Corrected_Target.set'],'filepath',[Pathname 'segmentsIO\']);
+%         
+%         
+%         EEG = pop_selectevent( tempEEG, 'type',3 ,'renametype','Standard','deleteevents','on','deleteepochs','on','invertepochs','off');
+%         EEG = pop_editset(EEG, 'setname',[subs{i_sub} '_' exp '_' conds{i_cond} '_Corrected_Standard']);
+%         EEG = pop_saveset( EEG, 'filename',[subs{i_sub} '_' exp '_' conds{i_cond} '_Corrected_Standard.set'],'filepath',[Pathname 'segmentsIO\']);
 
-exp = 'Skateboard';
-
-% loading regulars first
-subs = {'100' '101' '102' '103' '104' '106' '108' '111'...
-  '112' '116' '117' '118' '119' '120'};
-%subs = {'100'}; %to test on just one sub
+        if is_goofy(i_sub) 
+		
+			%this strcmp is how to compare strings 
+			% || means or, but | may be the right sympol (i always forget)
 
- nsubs = length(subs);
-%conds = {'P_CW';'P_CCW'; 'NP_CW'; 'NP_CCW'};
-conds =  {'P_CCW';'NP_CW'}; %facing OUTSIDE for regulars
-conds2 = {'facing_out1'; 'facing_out2'};
+			if strcmp(conds{i_cond},'P_CCW') || strcmp(conds{i_cond},'NP_CW')
+				ThisIsTheName = [subs{i_sub} '_' exp '_' new_conds{1} '_Corrected_Standard.set']
+            else
```

Comment:
from my view it looks like the indenting is not algined with the correct if statements, but I'm not sure that is the problem

Reply:
I don't think so either